src/NUnitConsole/nunit3-console/ConsoleRunner.cs

```diff
@@ -354,14 +347,16 @@ private void DisplayTestFilters()
             }
         }
 
-        private void RedirectErrorOutputAsRequested()
+        private ExtendedTextWriter CreateErrorWriter()
         {
             if (_options.ErrFileSpecified)
             {
                 var errorStreamWriter = new StreamWriter(Path.Combine(_workDirectory, _options.ErrFile));
                 errorStreamWriter.AutoFlush = true;
-                _errorWriter = errorStreamWriter;
+                return new ExtendedTextWrapper(errorStreamWriter);
             }
+
+            return _outWriter;
```

Comment:
Won't this change the default output for errors from `stderr` to `stdout`? Meaning if I'm redirecting the two streams separately (in cmd, outside of NUnit) - it'll be a change in behaviour?